clang/test/CIR/CodeGen/c89.c

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -std=c89 -emit-cir %s -o %t.cir
```

Comment:
Please rename the test to `c89-implicit-ret-type.c`, we can rename back later if we add more test here.

Reply:
Renamed to `c89-implicit-int.c` since it's also testing for implicit int on variable declarations, not only function return types.